Data/Entities/LogExercise.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace FitnessTracker.Data.Entities
+{
+    public class LogExercise
+    {
+        public int Id { get; set; }
+        public Log Log { get; set; }
+
+        public Exercise Exercise { get; set; }
```

Comment:
I think you're going to need the foreign keys for Log and Exercise not just the entity reference

Reply:
fixed by adding LogId and ExerciseId as FK's